pkg/auth/rbac.go

```diff
@@ -17,17 +17,28 @@
 package auth
 
 import (
+	"bytes"
+	"context"
+	"encoding/binary"
 	"fmt"
 	"net"
 	"strings"
 
+	"github.com/cilium/ebpf/ringbuf"
+
 	"kmesh.net/kmesh/api/v2/workloadapi"
 	"kmesh.net/kmesh/api/v2/workloadapi/security"
+	"kmesh.net/kmesh/pkg/bpf"
+	"kmesh.net/kmesh/pkg/controller/common/cache"
 	"kmesh.net/kmesh/pkg/logger"
 )
 
 const (
-	SPIFFE_PREFIX = "spiffe://"
+	SPIFFE_PREFIX     = "spiffe://"
+	MSG_TYPE_IPV4     = 0
+	MSG_TYPE_IPV6     = 1
+	IPV4_TUPLE_LENGTH = 12
+	MSG_LEN           = 40
```

Comment:
can you add some comments about these number, its hard for people without strong knowledge of the network

Reply:
done